tests/test_crosstalk.py

```diff
@@ -188,12 +188,18 @@ def testDirectAPI(self):
         ratios = measure.extractCrosstalkRatios(self.exposure, threshold=self.value - 1)
         coeff, coeffErr, coeffNum = measure.measureCrosstalkCoefficients(ratios)
         self.checkCoefficients(coeff, coeffErr, coeffNum)
-        subtractCrosstalk(self.exposure, minPixelToMask=self.value - 1,
-                          crosstalkStr=self.crosstalkStr)
+        calib = CrosstalkCalib()
+        calib.coeffs = coeff.transpose()
+        calib.coeffErr = coeffErr.transpose()
+        calib.coeffNum = coeffNum.transpose()
```

Comment:
There is a moderately terrifying amount of transposing happening here. If it is happening in the test I assume it is happening elsewhere too, does it affect runtime significantly?

Reply:
Moving MeasureCrosstalkTask to cp_pipe, including this test, is on my list as DM-24760.  The transpose here is a result of me normalizing the crosstalk coefficient order everywhere except there.  The only place a transpose should now be happening is in directly using the output of MeasureCrosstalkTask, and I believe this test is the only place that happens.

Reply:
Oh that's interesting, it hadn't occurred to me to have **measure** crosstalk task somewhere else, but I agree, that's where it "should" ultimately land. Sounds good.